src/core/send-commercial-metrics.ts

```diff
@@ -143,6 +143,21 @@ const getOfflineCount = (): Metric[] =>
 		  ]
 		: [];
 
+/**
+ * Measures added with @guardian/libs’s startPerformanceMeasure API
+ * that are assigned to the 'dotcom' team.
+ *
+ * Allows for more granular monitoring of web page performance.
+ */
+const getDotcomPerformanceMeasures = (): Metric[] =>
+	window.performance
+		.getEntriesByType('measure')
+		.filter(({ name }) => name.startsWith('doctom:'))
+		.map(({ name, duration }) => ({
+			name: name.replace(':', '_'),
```

Comment:
We should probably export this separator from libs

Reply:
I thought about this some more, and I’m wondering if we should use the “detail” key of performance measures for this, actually.

As mentioned in https://github.com/guardian/csnx/pull/756#pullrequestreview-1554250708, it would probably be best to have a deserialiser live in `@guardian/libs`… in the future